appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/impl/PayaraMicroImpl.java

```diff
@@ -1746,7 +1746,7 @@ private void deployAll() throws GlassFishException {
 
     private void addShutdownHook() {
         Runtime.getRuntime().addShutdownHook(new Thread(
-                "GlassFish Shutdown Hook") {
+                "Micro Shutdown Hook") {
```

Comment:
If we're fixing the name, I'd suggest this instead
```suggestion
                "Payara Micro Shutdown Hook") {
```